<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourWindow</class>
 <widget class="QWidget" name="ContourWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contour</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Data</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="1">
            <widget class="QPushButton" name="importDataButton">
             <property name="text">
              <string>Import Data</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="fileEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTableView" name="tableView"/>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Random</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Rows</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="rowsSpinbox">
             <property name="minimum">
              <number>3</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Preview</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
